docs/assets/includes/coverage-ignore.md

```diff
@@ -0,0 +1 @@
+To exclude files from coverage analysis only, you must ignore them directly in the tool you're using to generate coverage reports and ensure that the reports you upload to Codacy don't include coverage information for those files.
```

Comment:
Wondering if we really need the **only**... 🤔 
(in "To exclude files from coverage analysis only")